spark/components/alerts/vanilla/alerts.stories.js

```diff
@@ -7,7 +7,7 @@ export default {
 export const info = () => {
   useEffect(() => {
     alerts();
-  }, []);
+  });
```

Comment:
Hey @bobwise why did you remove the [] I was passing as a second argument into useEffect here?

Reply:
what does this array do?

Reply:
It makes it so the alerts function only runs once and that our effect doesn’t depend on any values from props or state, so it never needs to re-run. @bobwise

Reply:
ahh ok I found it in the docs. I didn't realize the empty array was a special use case. I'll put it back.